README.md

```diff
@@ -1,4 +1,4 @@
-# Figma Block for WordPress
+# Embed Block for Figma for WordPress
```

Comment:
Wondering if we just drop the `for WordPress` part. A little wordy otherwise to say `for Figma for WordPress`